nemo_aligner/data/nlp/builders.py

```diff
@@ -327,7 +328,7 @@ def build_dataloader(
     # Megatron sampler
     if hasattr(cfg.model.data, "dataloader_type") and cfg.model.data.dataloader_type is not None:
         if cfg.model.data.dataloader_type == "single":
-            cls = MegatronPretrainingBatchSampler if load_gbs else MegatronPretrainingSampler
+            cls = MegatronPretrainingRandomBatchSampler if load_gbs else MegatronPretrainingSampler
```

Comment:
Ok in that case can you make sure this is mentioned as a breaking change? Just so that users understand why some runs may now look different with the same settings as before.

Reply:
I have amended the changelog to notify people of this breaking change